pkg/env/env.go

```diff
@@ -45,6 +45,7 @@ type Interface interface {
 	Zones(vmSize string) ([]string, error)
 	ACRResourceID() string
 	ACRName() string
+	ACRDomain() string
```

Comment:
can ACRDomain() *replace* ACRName()?

Reply:
`ACRName` is currently used by itself without the domain in a `ClusterSpec` in one location: https://github.com/Azure/ARO-RP/blob/2e1fb1b00a9c8eb5b19af287b10ece6172747f66/pkg/operator/deploy/deploy.go#L149

Reply:
Yes - I thought `ACRName` was still needed in the operator, but I was mistaken.